### DISCUSSION TITLE
Question of selecting items when treeview clicks on the blank space

### DISCUSSION THREAD

**Discussion by pyking-wang** [AvaloniaUI/Avalonia#13059] [Q&A] [answered]
When I click on the blank space of the treeview control, it only remembers the previous node. 
How to correctly make Selectitem null？Thank you.
![image](https://github.com/AvaloniaUI/Avalonia/assets/76192669/1317ab2a-0ef9-4c2c-94e9-8aef857b5bd1)

**Comment by timunie** [comment, marked as answer]
handle pointer pressed event and check e.OriginalSource for beeing a TreeViewItem. If not, set SelectedItem = null

**Reply by pyking-wang** [reply]
How to handle pointer pressed events in the ViewModel layer?It cannot pass through binding.

**Reply by timunie** [reply]
No, you need code behind or an attached property / behavior

**Reply by pyking-wang** [reply]
ok，thanks

**Comment by pyking-wang** [comment]
The PointerPressed in treeview is only valid when clicking on a blank space, and is not valid when clicking on a node.
![image](https://github.com/AvaloniaUI/Avalonia/assets/76192669/88cd23cc-6f6a-4f69-b00e-1b071c13ebb0)
